From e05814549d537f05071518c31a6f59a755869636 Mon Sep 17 00:00:00 2001
From: Xiaowen Liu <b37945@freescale.com>
Date: Thu, 30 May 2013 15:19:38 +0800
Subject: [PATCH 19/50] ENGR00264642 change i frame interval from 15s to 1s.

change i frame interval from 15s to 1s to recover
the mess show on sink quickly.
because VPU does not support macro block refresh
mechanism well and can not recover the mess show.

Signed-off-by: Xiaowen Liu <b37945@freescale.com>
---
 media/libstagefright/wifi-display/source/Converter.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/media/libstagefright/wifi-display/source/Converter.cpp b/media/libstagefright/wifi-display/source/Converter.cpp
index 753b3ec..4482c06 100644
--- a/media/libstagefright/wifi-display/source/Converter.cpp
+++ b/media/libstagefright/wifi-display/source/Converter.cpp
@@ -195,7 +195,7 @@ status_t Converter::initEncoder() {
         mOutputFormat->setInt32("bitrate", videoBitrate);
         mOutputFormat->setInt32("bitrate-mode", OMX_Video_ControlRateConstant);
         mOutputFormat->setInt32("frame-rate", 30);
-        mOutputFormat->setInt32("i-frame-interval", 15);  // Iframes every 15 secs
+        mOutputFormat->setInt32("i-frame-interval", 1);  // Iframes every 15 secs
 
         // Configure encoder to use intra macroblock refresh mode
         mOutputFormat->setInt32("intra-refresh-mode", OMX_VIDEO_IntraRefreshCyclic);
-- 
1.8.0

